package.json

```diff
@@ -8,6 +8,7 @@
     "node-fetch": "^2.6.1",
     "pino": "^6.11.3",
     "sqlite-parser": "^1.0.1",
+    "yaml": "^1.10.2",
```

Comment:
why yaml?

Reply:
because it's good to write down declarative things, such as configurations or schemas definitions. 
This was just a playground to open the door to other ways to define the schemas rather than using an HTTP endpoint